engine_modules/corporation/models.py

```diff
@@ -92,20 +96,39 @@ class Meta:
 
 	base_corporation_slug = models.CharField(max_length=20)
 	game = models.ForeignKey(Game)
-	assets = models.SmallIntegerField()
-	historic_market = models.ForeignKey(Market)
+	assets = models.SmallIntegerField(default=0)
+	market_assets = models.SmallIntegerField()
```

Comment:
This was strictly for migrations purposes. I do believe it is moot, now. Consider it removed.